scripts/migration/migrate_deleted_wikis.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website.models import NodeLog
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+def get_targets():
+    return NodeLog.find(Q('action', 'eq', NodeLog.WIKI_DELETED))
+
+def migrate(targets, dry_run=True):
+    # iterate over targets
+    for log in targets:
+        node = log.node
+        versions = node.wiki_pages_versions
+        current = node.wiki_pages_current
+        updated_versions = {}
+        for wiki in versions:
+            if wiki in current:
+                updated_versions[wiki] = versions[wiki]
+        with TokuTransaction():
+            node.wiki_pages_versions = updated_versions
+            node.save()
+            if dry_run:
```

Comment:
This wasn't exactly what Steve said. The if dry_run should be in the main() function's `with TokuTransaction()` otherwise it'll only do one pass through the loop and stop.